beets/autotag/hooks.py

```diff
@@ -157,9 +158,10 @@ def __init__(self, title, track_id, artist=None, artist_id=None,
                  medium_total=None, artist_sort=None, disctitle=None,
                  artist_credit=None, data_source=None, data_url=None,
                  media=None, lyricist=None, composer=None, composer_sort=None,
-                 arranger=None, track_alt=None):
+                 arranger=None, track_alt=None, work_id=None):
```

Comment:
Thanks, did this and made the other suggested update but no luck. Any other thoughts?